### ISSUE TITLE
Better footnotes UX

### ISSUE THREAD

**Issue by srid** [srid/emanote#110]
Currently, footnotes are rendered very basically. See a demo here: https://note.ema.srid.ca/demo/markdown

We could spice things up here. For inspiration, see https://jamesclear.com/first-principles

Could use JavaScript for this work. Should be [accessible](http://kb.daisy.org/publishing/docs/html/roles.html), and work on mobile.

**Comment by srid** [comment]
Tip: code for this would go here: 

https://github.com/srid/emanote/blob/b04c725e8b70801e5abed2a549e58470b61b3f9d/default/templates/components/pandoc.tpl#L54-L77

**Comment by Mohib834** [comment]
Hi @srid, I would like to work on this one.

**Comment by kukimik** [comment]
@Mohib834 Are you working on this?

As an alternative maybe we could add backlinks to footnotes, like it is done for example [here](https://bigdjrp.github.io/quick-kramdown-demo/) or [here](https://www.sitepoint.com/accessible-footnotes-css/). I hate having to scroll up and search through the text after clicking on a footnote link.

**Comment by srid** [comment]
I don't think anyone is working on this. Feel free to prototype something!

**Comment by srid** [comment]
Agreed, backlinks (as in that arrow clicking which takes you to the footnote ref; not "backlinks" in the zettelkasten sense) would also be a good enough solution.
